homeworks/denis.yeromenko_Denis-Yeromenko/homework_6/homework_6_1.js

```diff
@@ -0,0 +1,38 @@
+const user = {};
+
+Object.defineProperty(user, 'fullName', {
+    get: () => this.fullName,
```

Comment:
arrow functions here will access global scope, avoid this

Reply:
fixed